sqe-dto/Transformation.cs

```diff
@@ -18,15 +18,15 @@ public class PlacementDTO
         public decimal rotate { get; set; }
 
         [Required]
-        public uint zIndex { get; set; }
+        public int zIndex { get; set; }
 
         [Required]
         public TranslateDTO translate { get; set; }
     }
 
     public class TranslateDTO
     {
-        public uint? x { get; set; }
-        public uint? y { get; set; }
+        public int? x { get; set; }
```

Comment:
Why are these nullable if we have isPlaced? Under what circumstances are these going to be null?